docs/release-notes/5.7.41-44.md

```diff
@@ -0,0 +1,66 @@
+# Percona Server for MySQL 5.7.41-44 (2023-)
+
+<style>1
+    table {
+        width=50%';
+        font-family: Chivo, Colfax, Franziska, Helvetica, Arial, sans-serif;
+    }
+    table td {
+        border: 0px;
+        padding: 8px;
+    }
+    tr:nth-child(even){
+        background-color:#f5f5f5
+    }
+    tr:hover {
+        background-color: #dddd;
+    }
+</style>
+
+| Release date | 2023 |
+|---|---|
+| Install instructions | [Install Percona Server for MySQL](https://www.percona.com/doc/percona-server/5.7/installation.html) |
+| Download this version | [Percona Server for MySQL](https://www.percona.com/downloads/Percona-Server-5.7/LATEST/)
+
+[Percona Server for MySQL](https://www.percona.com/software/mysql-database/percona-server) 5.7.41-44
+includes all the features and bug fixes available in [MySQL 5.7.41 Community Edition](https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-41.html) in addition to enterprise-grade features developed by Percona.
+
+*Percona Server for MySQL* is a free, fully compatible, enhanced, and open
+source drop-in replacement for any *MySQL* database. It provides superior
+performance, scalability, and instrumentation.
+
+*Percona Server for MySQL* is trusted by thousands of enterprises to provide
+better performance and concurrency for their most demanding workloads. It
+delivers more value to *MySQL* server users with optimized performance,
+greater performance scalability and availability, enhanced backups, and
+increased visibility.
+
+For paid [support](https://www.percona.com/services/support), [managed services](https://www.percona.com/services/managed-services) or [consulting services](https://www.percona.com/services/consulting), contact [Percona Sales](https://www.percona.com/about-percona/contact)
+
+For [training](https://www.percona.com/training), contact [Percona Training - Start learning now](https://learn.percona.com/contact-me)
+
+## Release highlights
+
+Improvements and bug fixes provided by Oracle for *MySQL* 5.7.41 and included in Percona Server for MySQL are the following:
+
+* Updated the linked OpenSSL library for MySQL Server to 1.1.s
+  
+* Updated the bundled zlib library to zlib 1.2.13. This zlib library version is the minimum supported.
+
+* While the SQL thread handled a transaction, issuing `STOP_SLAVE_SQL_THREAD` caused replication to stop immediately instead of waiting for the event group to complete before the shutdown.
+  
+* When the `--single-transaction` and `--set-gtid-purged=ON` options were both used the mysqldump backup of the data and GTIDs were inconsistent. Now at the beginning of the dump, a `FLUSH_TABLES_WITH_READ_LOCK` and `GTID_EXECUTED` are performed. These operations ensure that the GTID value is consistent with the mysqldump snapshot. 
+
+Find the full list of bug fixes and changes in the [MySQL 5.7.40 Release Notes](https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-41.html).
+
+There are no bug fixes for this version.
```

Comment:
We have one bug fix - https://jira.percona.com/browse/PS-7538

Reply:
Added.